docs/solution/analytics/index.md

```diff
@@ -5,15 +5,26 @@
 CrateDB provides real-time analytics on raw data stored for the long term.
 :::
 
-In all domains of real-time analytics where you absolutely must have access to all
-the records, and can't live with any down-sampled variants, because records are
-unique, and need to be accounted for within your analytics queries.
-
-If you find yourself in such a situation, you need a storage system which
-manages all the high-volume data in its hot zone, to be available right on
-your fingertips, for live querying. Batch jobs to roll up raw data into
-analytical results are not an option, because users' queries are too
-individual, so you need to run them on real data in real time.
+CrateDB eliminates the trade-off between data accessibility and storage costs
+by keeping all high-volume raw data in the hot zone without requiring
+downsampling or aggregation. Unlike traditional systems that force you to
+choose between real-time query capabilities and long-term retention,
+CrateDB handles billions of unique records while maintaining fast query
```

Comment:
```suggestion
CrateDB handles billions of records while maintaining fast query
```

Reply:
Thanks. This and your other suggestions have been added with d05341865d299.